### PR TITLE
separating delegated and all validators views

### PR DESCRIPTION
**Pull request by blackpanther1881**
Updates
1.  separating delegated and all validators views.
2.  separating active and inactive validators in the redelegation validators dropdown.
3.  Keystore file change. After the first txn changing Keystore file without login again.
4. Added foundation node warning.

### CODE REVIEW

File: src/actions/tokenPrice.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import Axios from 'axios';
-
 import {
     TOKEN_PRICE_FETCH_ERROR,
     TOKEN_PRICE_FETCH_SUCCESS,
 } from "../constants/tokenPrice";
+import CoinGecko from 'coingecko-api';
+
 
-const PROXY_API = process.env.REACT_APP_PROXY_API;
```
Comment (puneet2019) [inline src/actions/tokenPrice.js:8]: can you update this in README.md?

**Review by puneet2019 — APPROVED** [review]

**Review by puneet2019 — APPROVED** [review]

**Review by puneet2019 — DISMISSED** [review]

File: src/containers/KeyStore/ModalDecryptKeystore/index.js [hunk 2]
```diff
@@ -284,6 +289,16 @@ const ModalDecryptKeyStore = (props) => {
                                         }
                                     </Card>
                                 </Accordion>
+                                {
+                                    !importMnemonic ?
+                                        <div className="buttons">
+                                            <p className="button-link"
+                                                onClick={handleUpdateKeystore}>
+                                                Change Keystore file
```
Comment (deepanshutr) [inline src/containers/KeyStore/ModalDecryptKeystore/index.js:297]: pick from messages

File: src/containers/Staking/Validators/DelegatedValidators/index.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+const DelegatedValidators = (props) => {
+    const [modalDelegate, setModalOpen] = useState();
+    const [validator, setValidator] = useState('');
+    const handleModal = (name, validator) => {
+        setModalOpen(name);
+        setValidator(validator);
+    };
+    const columns = [{
+        name: 'validator',
```
Comment (deepanshutr) [inline src/containers/Staking/Validators/DelegatedValidators/index.js:20]: all string from messages

**Review by deepanshutr — DISMISSED** [review]
Reviewed.

**Review by deepanshutr — APPROVED** [review]

**Pull request merged**